xcode/Subconscious/Shared/Models/Petname.swift

```diff
@@ -7,6 +7,12 @@
 
 import Foundation
 
+enum PetnameResolutionStatus: Equatable, Hashable, Codable {
```

Comment:
Qs:

- Should this be a general enum, similar to `LoadingState`?
  - Is this the same semantics as `LoadingState`? It's ok if subtly different.
  - If different, still worth generalizing this I bet. We have a lot of places that have the same network resolution lifecycle.
- Is there a resolved success/failure? (`resolved(Result)`)

Non-blocking.

Reply:
I dropped the `Petname` prefix and extracted the type and added a `Cid` payload to `resolved` since I currently view the definition of resolution as "take this address and give me the CID on the other end".